src/Stryker.Core/Stryker.Core/Initialisation/ProjectFileReader.cs

```diff
@@ -13,7 +13,6 @@ namespace Stryker.Core.Initialisation
     public interface IProjectFileReader
     {
         ProjectAnalyzerResult AnalyzeProject(string projectFilepath, string solutionFilePath);
-        IEnumerable<string> FindSharedProjects(XDocument document);
```

Comment:
Why remove from the interface but not from the implementation? If it's not needed anymore I'd like the method to be deleted as well from the implementation.

Reply:
The method is needed, but is never used through the interface. Definitely a design smell. I am working on a better design, but it breaks a lot of tests.

Reply:
I migrated the method to `InputFileResolver`class.